test/.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": ["next"],
+  "rules": {
+    "import/no-anonymous-default-export": "off",
```

Comment:
@timneutkens I noticed you enabled this and updated a ton of unnamed exports in tests. Can do the same here if you like, but didn't want to duplicate efforts.

Reply:
It's fine to disable it for this. The tests can be kept as-is.